modules/codegen/src/main/scala/com/twilio/guardrail/extract/VendorExtension.scala

```diff
@@ -66,6 +67,8 @@ object VendorExtension {
     implicit val defaultVendorExtensibleCookieParameter: VendorExtensible[CookieParameter] =
       build[CookieParameter](m => key => Option(m.getExtensions).map(_.get(key)))
 
+    implicit val defaultVendorExtensibleSecurityScheme: VendorExtensible[SecurityScheme] =
+      build[SecurityScheme](m => key => Option(m.getExtensions).map(_.get(key)))
```

Comment:
I hesitate to ask, but what happens if `get(key)` fails? `null` or `Exception`?

Reply:
It's just a `java.util.Map`, I believe, so `null`.  At any rate, that's how all the other `VendorExtensible` impls work, so I assumed you believed `null` was ok there, and that something else in the extension extraction would take care of it?  Seeing as the vendor extensions actually work and never blow up, seems like it must be ok.

Reply:
So these should actually all be `flatMap` on `v => Option(v.get(key))`?

Reply:
otherwise we're relying on the trinary state of somehow ending up with an empty security model?

Reply:
Literally all the other `VendorExtensible` implementations do exactly the same thing.  The reason why it works is because the `PartialFunction` impls for `Extractable` all match on types that are not `Object`, and so they'll never get a `null` value passed to them.

Reply:
Essentially it works like this:
```
scala> val xs = List(1, "a", null, true)
xs: List[Any] = List(1, a, null, true)

scala> val pf: PartialFunction[Any, String] = {
     | case s: String => s
     | }
pf: PartialFunction[Any,String] = <function1>

scala> xs.map(x => scala.util.Try(pf(x)).toOption)
res0: List[Option[String]] = List(None, Some(a), None, None)
```

Reply:
Yep, there's just a completely different set of neurons firing when reading other people's PRs, even if it's code I originally wrote or maintained. Just looking at continuing to improve the codebase

Reply:
Yeah... I think there's room for improvement here; if someone ever changes `Extractable` to not be based on partial functions (or something) or tries to remove the `Try()` wrapper on calling the extractor, then things might stop working.  But I think we can leave that for antoher day...

Reply:
Fair. I'll create an issue to track